src/graphql/schema/uuid/abstract-uuid/resolvers.ts

```diff
@@ -38,3 +59,19 @@ export const resolvers: UuidResolvers = {
     },
   },
 }
+
+export async function toThreadPayload(
+  serlo: SerloDataSource,
+  firstCommentId: number
+): Promise<ThreadPayload> {
+  const firstComment = (await serlo.getUuid<CommentPayload>({
+    id: firstCommentId,
+  })) as CommentPayload
+  const remainingComments = Promise.all(
+    firstComment.childrenIds.map((id) => serlo.getUuid<CommentPayload>({ id }))
+  )
+  return {
+    __typename: DiscriminatorType.Thread,
+    commentPayloads: [firstComment, remainingComments],
```

Comment:
```suggestion
    commentPayloads: [firstComment, remainingComments.filter(isDefined)],
```
You need to run `git checkout revisions-add-to-repository -- src/graphql/schema/utils.ts` first. This gives you a function `isDefined` which is a type guard returning true when a value is not `null` or `undefined` (So you can add `.filter(isDefined)` instead of `.filter(v => v !== null) as ...Payload[]` and the type casting in the end is not necessary)